app/models/contact.rb

```diff
@@ -26,6 +26,13 @@ class Contact < ApplicationRecord
   include Avatarable
   include AvailabilityStatusable
   include Labelable
+  include PgSearch::Model
+  include MultiSearchableHelpers
```

Comment:
ensure to add an after update method which will reindex the conversation records if contact email phone number etc gets updated